dev-docs/howtoguides/how_to_setup_windows_machine_for_wsl_testing.md

```diff
@@ -0,0 +1,158 @@
+# How to setup Windows machine for WSL testing
+
+To run our integration tests on WSL instances, we need to first setup a Windows machine
+to allows us to launch the WSL instances. Not only that, we also need to setup OpenSSH
+server to allow us to run ssh commands on the Windows machine.
+
+This how to guide will explain all the necessary steps for the setup, from launching the image
+from Azure to testing if WSL is correctly working on the machine.
+
+## Launching the image on Azure
+
+When launching the Windows instance on Azure, there a few configurations that need to be performed,
+otherwise we will not be able to launch WSL instances.
+
+First of all, name the machine as `wsl-test`. Currently, this is the name we expect on the
+integration tests.
+
+After that, select the `Windows 11` image and mark the `Security Type` as *Standard*.
+![windows_secuirty_type](windows-security-type.png)
+
+Set an username and password for the VM. Please set the username as *ubuntu*. 
+We will use this account setting to manually login into the machine to run some commands there directly.
+![windows_secuirty_type](windows-ssh-setup.png)
```

Comment:
```suggestion
![windows_ssh_setup](windows-ssh-setup.png)
```

Reply:
I am leaving this blank as @s-makin suggested